package.json

```diff
@@ -10,22 +10,25 @@
   "scripts": {
     "prepare": "typesync && lerna run --concurrency 8 --stream prepare",
     "build": "lerna run build",
-    "build:ci": "lerna run --since $(git merge-base $CIRCLE_BRANCH origin/master) build:ci",
+    "build:ci:incremental": "yarn build:ci --since $(git merge-base $CIRCLE_BRANCH origin/master)",
+    "build:ci": "lerna run build:ci",
```

Comment:
What's the reason for using `yarn` on the incremental build but `lerna` on the master build?

Reply:
`yarn` runs `build:ci` defined in the same `package.json`. In this case, `build:ci:incremental` will trigger `build:ci` aka `lerna run build:ci` with an additional `--since` flag.